test/unit/user_test.rb

```diff
@@ -93,6 +93,91 @@ class UserTest < ActiveSupport::TestCase
     end
   end
 
+  test 'send_daily_triage! sends email after preferred email_time_of_day' do
+    user  = users(:morning_person)
+    repo  = repos(:rails_rails)
+
+    issue = repo.issues.new
+    issue.title           = "Foo Bar"
+    issue.url             = "http://schneems.com"
+    issue.last_touched_at = 2.days.ago
+    issue.state           = 'open'
+    issue.html_url        = "http://schneems.com"
+    issue.number          = (repo.issues.last.try(:number) || 0) + 1
+    issue.save
```

Comment:
It is not required to create the issue with all attributes set as this test is not particularly testing any of those values. Eg, It can be `repo.issues.create(title: 'Foo Bar')` or `issue = repo.issues.new(title: 'Foo Bar'); issue.save`